### ISSUE TITLE
data preprocessing steps to add (windowing, sampling, etc.)

### ISSUE THREAD

**Issue by kshannon** [kshannon/intracranial-hemorrhage-detection#16]
brain W:80 L:40
subdural W:130-300 L:50-100
stroke W:8 L:32 or W:40 L:40 3
soft tissues: W:350–400 L:20–60 4

**Comment by kshannon** [comment]
**Windowing Strategy:** After reading a few papers, suggest we try using these as the three RGB channels:
1. WL:40, WW:40 (tissue window)
2. WL:50, WW:100 (brain window)
3. WL:60, WW:40 (blood window)

**Validation Strategy:** Validation dataset should be a balances class 1/0 set whose subtype distributions match the training set  roughly, (might be tough with Epidural TH subtype).

**Comment by kshannon** [comment]
windowing strategy has been implemented, going to try it out on resnet50.

**Comment by kshannon** [comment]
@utcsox @tonyreina refactored a bunch of code and got a lot of stuff working, switched over to 3 channel windowing, toying around with a custom loss function, downsampling, and added resnet50. We have a pretty good base for experimentation. I also added new class balanced csv files, with clean data.

![Screenshot from 2019-10-11 02-23-54](https://user-images.githubusercontent.com/1323888/66642051-c574c580-ebd0-11e9-8d73-d12bde120649.png)
